workflows/theiameta/wf_theiameta_illumina_pe.wdl

```diff
@@ -232,6 +232,12 @@ workflow theiameta_illumina_pe {
     String bbduk_docker = read_QC_trim.bbduk_docker
     # Read QC - Read stats
     Float? average_read_length = read_QC_trim.average_read_length
+    # MIDAS outputs
+    String? midas_primary_genus = read_QC_trim.midas_primary_genus
+    String? midas_secondary_genus = read_QC_trim.midas_secondary_genus
+    Float? midas_secondary_genus_abundance = read_QC_trim.midas_secondary_genus_abundance
```

Comment:
Although these outputs are available, they don't make much sense from a metagenomics point of view. From a community you're expected to have a lot of stuff in there, depending on the complexity of the sample. I'm unsure if they should be included in TheiaMeta... @sage-wright do you have opinions regarding those?

Reply:
Didn't you request all these outputs int he original issue? https://github.com/theiagen/public_health_bioinformatics/issues/308

Reply:
Yes, but thinking about it conceptually now it might induce people in error when using TheiaMeta to have outputs that are tailored to perform QC on WGS data. Better to adjust now than to merge and then having to create a new issue. Just waiting on folks opinions on this. Tagging @AndrewLangvt @emily-smith1 @michellescribner for visibility!

Reply:
We probably want midas_primary_genus too?, I'm happy to remove the other outputs and just have the midas_report.tsv. The request for Midas Dbs below, if you can raise that with @AndrewLangvt. Id see that as another issue we could add to the list

Reply:
I agree that secondary genus information may not be the most accurate for a metagenomic sample. I think we should keep both the `midas_report` and `midas_primary_genus` outputs here.

Reply:
ok Ill update and just keep those outputs, thanks for the input!